Swarm/src/config/envs/docker.ts

```diff
@@ -51,7 +51,8 @@ export function createDockerConfig() {
             transactions: {
                 endpointPrefix: "transactions",
                 host: "0.0.0.0",
-                port: 8089
+                port: 8089,
+                costUpdatedInterval: 60
```

Comment:
is this a bug, my diff doesnt show any changes in Swarm

Reply:
it does appear to be a bug! I'm not seeing a diff here either